src/components/CookieWarning.js

```diff
@@ -13,17 +14,55 @@ import { authStore } from 'src/libs/state'
 export const cookiesAcceptedKey = 'cookiesAccepted'
 
 const CookieWarning = () => {
+  const animTime = 0.3
   const signal = useCancellation()
+  const [showWarning, setShowWarning] = useState(false)
+  const [opacity, setOpacity] = useState(0)
   const { cookiesAccepted } = useStore(authStore)
+  const timeout = useRef()
 
   const acceptCookies = acceptedCookies => {
     authStore.update(_.set('cookiesAccepted', acceptedCookies))
   }
 
+  useEffect(() => {
+    if (cookiesAccepted === undefined) {
```

Comment:
When is `cookiesAccepted` expected to be undefined? (Asking since this PR changes the state for non-signed in users from undefined to false).

Reply:
cookiesAccepted is undefined when it is first declared, then set to be false after we check the gapi and see the user status. I'm using "undefined" to mean "still loading"

Reply:
Ah, I see. Thanks for the explanation.